test/test/addressbook-level2/runtests.bat

```diff
@@ -0,0 +1,21 @@
+@ECHO OFF
```

Comment:
Did you intend to change this file? I don't see any differences aside from the omitted new line at the end of the file (which is not supposed to be removed). Same goes for the other irrelevant files changed.